attributes/shadows.css

```diff
@@ -1,4 +1,4 @@
-@import url('../vendor/photon-colors.css');
+@import url('~photon-colors/photon-colors.css');

```

Comment:
:+1:

Reply:
Wondering what do you think of the attributes directory for "not quite components" but "not quite base.css" type css?